awx/conf/apps.py

```diff
@@ -12,6 +14,9 @@ class ConfConfig(AppConfig):
 
     def ready(self):
         self.module.autodiscover()
-        from .settings import SettingsWrapper
 
-        SettingsWrapper.initialize()
+        if not set(sys.argv) & {'migrate', 'check_migrations'}:
```

Comment:
If you're filing follow issues, then let's get one to remove some hacks in `awx/conf/settings.py`. With this in place, I don't feel like that should ever be checking `sys.argv`. I quickly searched, and didn't find any sign of that file in the current diff.

Reply:
I'm not sure I know what hacks you're referring to.

Reply:
https://github.com/ansible/awx/blob/4e97492ed624eeb847c96e09d65478726eb81036/awx/conf/settings.py#L84

Reply:
Right, gotcha.  Yep, I'll file the ticket.